src/NumericalAlgorithms/DiscontinuousGalerkin/NormalDotFlux.hpp

```diff
@@ -74,6 +74,53 @@ auto normal_dot_flux(
 }
 /// @}
 
+/// @{
+/*!
+ * \brief Multiplies a surface normal covector with a flux tensor or variables
+ *
+ * \details
+ * Returns the outer product $n_j v_{k\ldots}$, where $n_j$ is the `normal` and
+ * $v_{k\ldots}$ is the `flux_tensor`.
```

Comment:
I believe "flux tensor" usually refers to `F`, as in the documentation of the preceding function in this file.

Reply:
Renamed to `rhs`